### ISSUE TITLE
QT4 build failure on osx

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CDrummond/cantata#467]
```
What steps will reproduce the problem?
1. Follow instructions for qt4 while using OS X 10.9

What is the expected output? What do you see instead?
Expectation: For the first make command to complete successfully
Reality: 
http://pastebin.com/9vU8qX2q

What version of the product are you using? On what operating system?
svn/trunk, but it happens on older versions as well

Please provide any additional information below.
Basically qxt causes the build to fail.  Performing the following changes to 
cantata-read-only/CMakeLists.txt to disable qxt will allow build to complete 
sucessfully:
http://pastebin.com/tctQVFYh

qt4 and cmake are both installed via homebrew, cmake finds qt4 automatically
```

Original issue reported on code.google.com by `Imaweasal` on 18 Apr 2014 at 7:20
